pkg/xen-tools/xen-start

```diff
@@ -74,6 +74,14 @@ xl create "$2" -p $EXTRA_ARGS || bail "xl created failed"
 
 # we may need to wait for domain to come online for us to manipulate it (timing out in under 30 sec)
 ID=$(domID "$1")
+
+if [ $# -eq 3 ]
+then
+  PRODUCT_NAME="OpenStack Compute"
+  # FIXME: it is not persist after VM reboot from inside
+  xenstore-write "/local/domain/$ID/bios-strings/system-product-name" "$PRODUCT_NAME" || echo "fail to set product name"
```

Comment:
Do you need this here and as the dmArgs? I would hope one place would be sufficient.

Reply:
Well, I cannot pass two-word option here as argument, so I pass just `overwrite` as the last arg. Inside xen-start I check for it and made write to xenstore.